docs/support/ios.rst

```diff
@@ -104,20 +104,30 @@ IOS requires that the banner use the EXT character (ASCII 3). This looks like a
     ...   f.write("banner motd {}\n".format(ext_char))
     ...   f.write("my banner test\n")
     ...   f.write("{}\n".format(ext_char))
-    ... 
+    ...
     >>> quit()
 
 Configure replace operations will reject a file with a banner unless it uses the ASCII character. Note, this likely also implies you cannot just copy-and-paste what you see on the screen.
 
 In vim insert, you can also type <ctrl>+V, release only the V, then type C
 
 
+File Operation Prompts
+_____
+
+By default IOS will prompt for confirmation on file operations. These prompts need to be disabled before the NAPALM-ios driver requires these to be disabled.
```

Comment:
Second sentence doesn't make sense?

Reply:
Hahaha, no, it really doesn't. I'll fix it...